### ISSUE TITLE
Potential stuck in generating flex trimap

### ISSUE THREAD

**Issue by WANGEOGEO** [Charch-630/FGI-Matting#5]
Hi,

During the training process, sometimes I found that my flex trimap generation step gets stuck forever. This issue takes place randomly which makes it hard to debug. Have you met similar problem during training?

And, I am wondering when will the training code be released? 

Thanks for answering in advance.

**Comment by Charch-630** [comment]
Yes, I use `try ... catch ...` to solve this problem. This is because sometimes `np.polyfit` may fail.

The training code won't be released, we only release the test code and demo.
